libraries/AP_HAL_ChibiOS/hwdef/CubeRedPrimary/hwdef.dat

```diff
@@ -0,0 +1,260 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H757
+
+# MCU class and specific type
+MCU STM32H7xx STM32H757xx
+
+# USB setup
+USB_VENDOR 0x2DAE # ONLY FOR USE BY HEX! NOBODY ELSE
+USB_PRODUCT 0x1059
+USB_STRING_MANUFACTURER "CubePilot"
+USB_STRING_PRODUCT "CubeRed"
+
+define CORE_CM7
+define SMPS_PWR
+
+define WATCHDOG_DISABLED
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1069
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 128
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
```

Comment:
Thoughts on 256k bootloader? Thinking Ethernet and maybe encryption in the future?

Reply:
Need to free up more space in Internal Flash to make that happen

Reply:
well.. yeah.. but does that mean you like the idea or not?